pkg/apis/controlplane/sets.go

```diff
@@ -38,10 +40,9 @@ func normalizeGroupMember(member *GroupMember) groupMemberKey {
 		b.WriteString(member.ExternalEntity.Namespace)
 		b.WriteString(delimiter)
 		b.WriteString(member.ExternalEntity.Name)
-	} else if len(member.IPs) != 0 {
-		for _, ip := range member.IPs {
-			b.Write(ip)
-		}
+	}
+	for _, ip := range member.IPs {
```

Comment:
member.IPs are always in the same order? i assume

Reply:
Yes. It will be the same order as `Pod.Status.PodIPs`, and I don't think it will be shuffled in runtime.